sdk_container/src/third_party/coreos-overlay/sys-libs/libsemanage/libsemanage-3.5.ebuild

```diff
@@ -79,38 +78,18 @@ multilib_src_compile() {
 		LIBDIR="${EPREFIX}/usr/$(get_libdir)" \
 		all
 
-	if multilib_is_native_abi; then
-		building_py() {
-			emake \
-				AR="$(tc-getAR)" \
-				CC="$(tc-getCC)" \
-				LIBDIR="${EPREFIX}/usr/$(get_libdir)" \
-				"$@"
-		}
-		python_foreach_impl building_py swigify
-		python_foreach_impl building_py pywrap
-	fi
+	dotmpfiles "${FILESDIR}/tmpfiles.d/libsemanage.conf"
```

Comment:
We already install this in `multilib_src_install`. Is this necessary do also do it in `multlib_src_compile`?